default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, lib, nix-gitignore, gnumake, pkgconfig, wget, unzip, gawk
+, sqlite, which, luaPackages, installShellFiles, makeWrapper
+}:
+let
+  luaLibs = with luaPackages; [ lua luasql-sqlite3 luautf8 ];
+in
+stdenv.mkDerivation rec {
+  pname   = "openrussian-cli";
+  version = "1.0.0";
+
+  src = nix-gitignore.gitignoreSource [] ./.;
+
+  nativeBuildInputs = [
+    gnumake pkgconfig wget unzip gawk sqlite which installShellFiles
+  ];
+
+  buildInputs = [ makeWrapper ] ++ luaLibs;
+
+  makeFlags = [
+    "LUA=${luaPackages.lua}/bin/lua"
+    "LUAC=${luaPackages.lua}/bin/luac"
+  ];
+
+  dontConfigure = true;
+
+  # Disable check as it's too slow.
+  # doCheck = true;
+
+  #This is needed even though it's the default for some reason.
+  checkTarget = "check";
+
+  # Can't use "make install" here
+  installPhase = ''
+    mkdir -p $out/bin $out/share/openrussian
+    cp openrussian-sqlite3.db $out/share/openrussian
+    cp openrussian $out/bin
+
+    wrapProgram $out/bin/openrussian \
+      --prefix LUA_PATH ';' "$LUA_PATH" \
+      --prefix LUA_CPATH ';' "$LUA_CPATH"
+
+    runHook postInstall
+  '';
+
+  postInstall = ''
+    installShellCompletion --bash --name openrussian ./openrussian-completion.bash
+    installManPage ./openrussian.1
+  '';
+
+  meta = with lib; {
+    homepage    = "https://github.com/rhaberkorn/openrussian-cli";
+    description = "Offline Console Russian Dictionary (based on openrussian.org)";
+    license     = licenses.gpl3;
```

Comment:
Licensing is a bit more complex - see "License" in README. But I don't know how this is handled in Nix. On Debian you could specify the license of every file individually.

Reply:
You can specify multiple licenses. See https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes

Fixed in https://github.com/rhaberkorn/openrussian-cli/commit/732172e2b49a1c468d9097cf937d8da83e6f061f.